packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -329,6 +330,16 @@ class WebView extends StatefulWidget {
   /// The default policy is [AutoMediaPlaybackPolicy.require_user_action_for_all_media_types].
   final AutoMediaPlaybackPolicy initialMediaPlaybackPolicy;
 
+  /// Whether to use `AndroidViewSurface` and `SurfaceAndroidViewController` to create the widget.
+  ///
+  /// This flag is temporary and is highly subject to removal. One should only
+  /// use this if they are willing to test [WebView] features that aren't
+  /// available in the standard version and acknowledges that it is possible
+  /// this can have slower performance and/or unpredictable bugs.
+  ///
+  /// Defaults to false.
+  final bool useExperimentalAndroidSurfaceView;
```

Comment:
What do you think about a separate `.dart` file that has this feature on by default?  This way, we have a `BUILD` target that automatically adds the metadata to the Android Manifest.  

@mehmetf do you have any preference about how to add this feature?

Reply:
I tried this out, but this file has a lot of private methods/constructors that needed to be copy pasted to the new file. I went with a different approach and decided just create a new `WebViewPlatform` implementation that extends `AndroidWebView`. So, now all a user needs to do is: `WebView.platform = SurfaceAndroidWebView()`. What are your thoughts on that approach.